include/SDL3/SDL_gpu.h

```diff
@@ -964,22 +964,22 @@ typedef struct SDL_GPUViewport
 
 typedef struct SDL_GPUTextureTransferInfo
 {
-    SDL_GPUTransferBuffer *transferBuffer;
-    Uint32 offset;      /* starting location of the image data */
-    Uint32 imagePitch;  /* number of pixels from one row to the next */
-    Uint32 imageHeight; /* number of rows from one layer/depth-slice to the next */
+    SDL_GPUTransferBuffer *transfer_buffer;
```

Comment:
buffer?

Reply:
"buffer" is a different conceptual type from "transfer buffer" in the API so I'm hesitant to lose the "transfer_" prefix in this case. Though maybe it's obvious from context since the type is GPUTransferBuffer and the struct itself has Transfer in the name...?

Reply:
Yeah, it seems clear from the type of the variable and the structure context.